build/set-env.ps1

```diff
@@ -6,38 +6,67 @@ $ErrorActionPreference = 'Stop'
 Write-Host "Setting up build environment variables"
 
 If ($Env:BUILD_BUILDNUMBER -eq $null) { $Env:BUILD_BUILDNUMBER ="0.0.1.0" }
+Write-Host "BUILD_BUILDNUMBER: $Env:BUILD_BUILDNUMBER"
```

Comment:
To reduce duplicating env variable names and `Write-Host` calls, it's probably easier to use `Get-ChildItem` to write out a list of environment variables:

```pwsh
Get-ChildItem -Path Env:/* -Include BUILD_BUILDNUMBER, BUILD_CONFIGURATION, PYTHON_VERSION  | Format-Table | Out-String | Write-Host
```

For readability in source, you can break up the above using `@()` to write out the array of environment variables instead:

```pwsh
Get-ChildItem -Path Env:/* -Include @(
    "BUILD_BUILDNUMBER",
    "BUILD_CONFIGURATION",
    "PYTHON_VERSION"
) | Format-Table | Out-String | Write-Host
```

Reply:
Thanks! Changing.